pages/en/lb3/Model-definition-JSON-file.md

```diff
@@ -404,6 +406,14 @@ For example:
         See <code>Id</code> property below.
       </td>
     </tr>
+    <tr>
+      <td>generated</td>
+      <td>No</td>
+      <td>Boolean</td>
+      <td>
+        <strong>For auto-migrate usage and the memory connector(id property only)</strong>. When it sets to <code>true</code>, the value of the property will be generated automatically with the default property type of the database. (e.g integer for MySQL and string for MongoDB)
```

Comment:
Not knowing the actual behavior, I'm a bit confused on this line:
```For auto-migrate usage and the memory connector(id property only)```

Does it mean it only works for:
- auto-migrate
- for memory connector, it only works for id property. For other connectors, there's no such restriction?

Reply:
Reworded and removed memory-connector part.